src/model.jl

```diff
@@ -227,8 +278,7 @@ function _build_model(m::JuMP.Model; case::Case, storage::Storage,
     end)
     if load_shed_enabled
         JuMP.@variable(m,
-            0 <= load_shed[i in 1:sets.num_load_bus, j in 1:interval_length]
-            <= bus_demand[sets.load_bus_idx[i], j],
+            load_shed[i in 1:sets.num_load_bus, j in 1:interval_length],
```

Comment:
It seems like with this change we can shed more load than we have available at a bus?

Reply:
I ended up constraining the `load_shed` variable separate from the variable creation. When demand flexibility is enabled, `load_shift_up` and `load_shift_dn` will have an effect on the upper bound of `load_shed`. I wasn't sure how flexible setting constraints within the variable creation was, so that's why I moved it to its own constraint.

Reply:
Here is how we modify variable bounds as we step through intervals: https://github.com/Breakthrough-Energy/REISE.jl/blob/develop/src/loop.jl#L86

Reply:
Thanks for pointing this out! I was just going to reply to your next comment suggesting that this is one of changes that I forgot to account for. Is it preferred to keep the `load_shed` variable the same as it was originally (include the constraints in the variable creation) and instead account for `load_shift_up` and `load_shift_dn` in `loop.jl` if demand flexibility is allowed?

Reply:
We need to make sure that load shedding is appropriately constrained in the first iteration of the loop (when we run `_build_model`), as well as appropriately updated in each iteration. We will have to do the same for flexibility (if present).

Reply:
@danielolsen I'm about to point the same thing out.

Reply:
I have updated `loop.jl`, so `load_shed` and the demand flexibility variables (`load_shift_up` and `load_shift_dn`) should be properly constrained in each iteration.